src/components/jobs/create/jobCreateScheduleDialog.ts

```diff
@@ -64,11 +65,50 @@ export class JobScheduleDialog {
             this.formState.schedule = true;
         }
 
-        if (!this.job.offset && this.job.offset !== 0) {
+        if ((!this.job.offset && this.job.offset !== 0) || this.job.offset < 0) {
```

Comment:
This check doesn't make a lot of sense:

If the job offset doesn't exist, AND the job offset !== 0.

That condition is a bit redundant.